commands/initLoggerCommands.go

```diff
@@ -0,0 +1,42 @@
+package commands
+
+import (
+	"fmt"
+
+	"github.com/ditrit/verdeter"
+	verdetermodels "github.com/ditrit/verdeter/models"
+)
+
+func initLoggerCommands(cfg *verdeter.VerdeterCommand) {
+
+	loggerModeKey := "logger.mode"
```

Comment:
Why do you have a variable for the key and not for the other config ?

Reply:
Now every config key use a variable